selfdrive/car/fw_query_definitions.py

```diff
@@ -9,6 +9,10 @@
 
 AddrType = Tuple[int, Optional[int]]
 EcuAddrBusType = Tuple[int, Optional[int], int]
+EcuAddrSubAddr = Tuple[int, int, Optional[int]]
+
+LIVE_FW_VERSIONS = Dict[AddrType, Set[bytes]]
+OFFLINE_FW_VERSIONS = Dict[str, Dict[EcuAddrSubAddr, List[bytes]]]
```

Comment:
Tried this ruff rule, but it didn't detect it for some reason: https://docs.astral.sh/ruff/rules/snake-case-type-alias/